masterlist.yaml

```diff
@@ -19,8 +97,56 @@ common:
         text: 'No limpiar las referencias ITM (iguales al master), ya que son intencionales y necesarias para que el Mod funcione. Sí es seguro restaurar las UDR (referencias borradas), pero no haga más que eso.'
       - lang: zh_CN
         text: '不干净。"脏"数据是故意的，这是mod需要的功能。'
+      - lang: da
+        text: 'Rengør ikke ITM-poster: de er forsætlige og krævede for at mod’en fungerer. Det er sikkert at gendanne poster, men gør ikke andet end dét.'
       - lang: ja
         text: 'IMTレコードはクリーンしないでください。これらは意図的に残されたデータであり、Modを機能させるために必要です。削除の取り消しは安全に行えますが、それ以外のことは行わないでください。'
+  - &patchAvailable
+    type: say
+    content:
+      - lang: en
+        text: 'This plugin conflicts with plugin "%1%", however a patch is available.'
+      - lang: da
+        text: 'Dette plugin er i konflikt med pluginnet “%1%”, men en lap er tilgængelig.'
+      - lang: ja
+        text: 'このプラグインは"%1%"と競合していますが、パッチが利用可能です。'
+  - &hasPluginPatch
+    type: warn
+    content:
+      - lang: en
+        text: "You seem to be using %1%, but you haven't enabled the %1% patch for this plugin: %2%"
+      - lang: da
+        text: "Du ser ud til at bruge %1%, men du bruger ikke “%1%”-lappen for dette plugin: %2%"
+      - lang: ja
+        text: "%1%が使用されていますが、このプラグイン用の%1%のパッチが有効になっていません: %2%"
+  - &requireSKSE64
```

Comment:
As far as I can see, this is the only `common` entry you've added that's actually used, why all the others?

Reply:
Haven't used YAML before so was mainly doing it to get an idea of how it was working.
And also because those entries were in FO4 masterlist which I assumed was more active and up to date.

Reply:
OK, well the FO4 masterlist is the most recent, but if an entry in the `common` section isn't referenced by its `&anchor` anywhere, it's not used and so is just padding the masterlist with noise, so it's better for it not to be there.